docs/source/deployment/deployment_guide.md

```diff
@@ -15,7 +15,6 @@ We also provide information to help you deploy to the following:
 * to [Prefect](prefect.md)
 * to [Kubeflow Workflows](kubeflow.md)
 * to [AWS Batch](aws_batch.md)
-* to [Databricks](databricks.md)
```

Comment:
I think we should still retain a link to the databricks deployment guide, but point through to its new location, right?